build_tools/github/build_minimal_windows_image.sh

```diff
@@ -22,10 +14,20 @@ cp $WHEEL_PATH $WHEEL_NAME
 # Dot the Python version for identyfing the base Docker image
 PYTHON_VERSION=$(echo ${PYTHON_VERSION:0:1}.${PYTHON_VERSION:1:2})
 
+# TODO: Remove when 3.11 images will be available for
+# windows (for now the docker image is tagged as 3.11-rc)
+# TODO: Remove when 3.11 images will be available for
+# scipy (for now only available from nightly anaconda index)
```

Comment:
I guess this can be removed now that SciPy and NumPy published wheels for Python 3.11 even if those two projects are still depending on Python 3.11 release candidates?